src/client/Microsoft.Identity.Client/ApiConfig/AcquireTokenByUsernamePasswordParameterBuilder.cs

```diff
@@ -37,6 +37,18 @@ internal AcquireTokenByUsernamePasswordParameterBuilder(IPublicClientApplication
                    .WithScopes(scopes).WithUsername(username).WithPassword(password);
         }
 
+        /// <summary>
+        /// Enables MSAL to read the federation metada for a wstrust exchange from a file instead of acquiring it from an endpoint
```

Comment:
```suggestion
        /// Enables MSAL to read the federation metadata for a wstrust exchange from a file instead of acquiring it from an endpoint
```